### PR TITLE
Rate limiter stats

### PR DESCRIPTION
**Pull request by ShanniLi**
r @Raynos @kriskowal @rf 
for issue #1138

### CODE REVIEW

File: node/lib/statsd.js [hunk 1]
```diff
@@ -120,7 +120,7 @@ TChannelStatsd.prototype.onStat = function onStat(stat) {
     var key = getKey(stat);
 
     if (stat.type === 'counter') {
-        return self.statsd.increment(key, stat.value);
+        return self.statsd.counter(key, stat.value);
```
Comment (Raynos) [inline node/lib/statsd.js]: do we really want this change ?

File: node/rate_limiter.js [hunk 2]
```diff
@@ -112,7 +112,7 @@ function refresh() {
     if (self.cycle === self.numOfBuckets) {
         self.channel.emitFastStat(self.channel.buildStat(
             'tchannel.rate-limiting.total-rps',
-            'gauge',
+            'counter',
             self.totalRequestCounter.rps,
```
Comment (Raynos) [inline node/rate_limiter.js]: I think this works because `refreshDelay` is 1000.

File: node/lib/statsd.js (hunk 1 above)
Comment (Raynos) [inline node/lib/statsd.js]: let's not make this unless needed.

Comment (ShanniLi) [inline node/lib/statsd.js]: fixed

**Comment by Raynos** [comment]
lgtm :D

**Pull request merged**